pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -159,6 +159,10 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, optio
 			TagSpecifications: []*ec2.LaunchTemplateTagSpecificationRequest{{
 				ResourceType: aws.String(ec2.ResourceTypeInstance),
 				Tags: []*ec2.Tag{
+					{
+						Key:   aws.String("Name"),
+						Value: aws.String(fmt.Sprintf("Karpenter/%s", ptr.StringValue(options.Cluster.Name))),
```

Comment:
Won't this collide, since we have many LTs per cluster?

Reply:
I don't think we can avoid collisions, maybe this should also have the provisioner as well, but it will still be the same name per instance per provisioner.

Reply:
It's also not per node. It's per-unique-hash-of-scheduling-constraints.

Reply:
I can see a reasonable argument that they should just all have the same name, but the philosophy behind AWS's naming concept is completely opaque to me.

Reply:
Nevermind I was totally confused about what entity this was tagging. All good.